examples/with-apollo/lib/initApollo.js

```diff
@@ -0,0 +1,37 @@
+import { ApolloClient, createNetworkInterface } from 'react-apollo'
```

Comment:
I hear you. I'm all for DRY, but developers are using these examples as boilerplate. Sharing a file would make one example dependent on another and could confuse people. It would also break the Create Next App CLI tool. For example, `create-next-app --example with-apollo-and-redux my-app` would then no longer work. Let's keep as is.